test_archiver/output_parser.py

```diff
@@ -548,6 +548,120 @@ def endElement(self, name):
             print("WARNING: ending unknown item '{}'".format(name))
         self._current_content = []
 
+class PhpJUnitOutputParser(XmlOutputParser):
+
+    def _report_test_run(self):
+        self.archiver.begin_test_run('php JUnit parser', None, 'phpunit', False, None)
+
+    def _handle_suites_from_class_name(self, class_name):
```

Comment:
This method is shared with pyunit parser. Maybe this should be defined in a super class for junit-style parser and these two parser then inherit from there.